hw11/Lesson11.py

```diff
@@ -0,0 +1,67 @@
+import datetime
+import inspect
+
+func_name = None
+
+def my_decorator_func(func):
+    def deco_func(*args, **kwargs):
+        global func_name
+        frame = inspect.currentframe()
```

Comment:
You shouldn't print the name of the wrapper function but the name of the decorated function - `func`. It is enough to `print(func)` to get its name printed with memory address, or `print(func.__name__)` to print only name